include/deal.II/base/mpi_remote_point_evaluation.h

```diff
@@ -366,6 +370,23 @@ namespace Utilities
       bool
       is_ready() const;
 
+      /**
+       * Return permuation needed for sending. If this is applied, evaluation
+       * results do not have to be sorted accoring to the receiving ranks
+       * during evaluate_and_process().
+       */
+      const std::vector<unsigned int> &
+      get_send_permutation() const;
+
+      /**
+       * Data is contiguously rank by rank. If no sorting is requested during
+       * evaluate_and_process(), this permutation allows to access values
+       * corresponding on evaluation point.
```

Comment:
I don't quite understand "corresponding on evaluation point". Would you mind looking this over again?

Reply:
I have extended the comments. Better?